study-builder/studies/pancan/snippets/question-describe.conf

```diff
@@ -0,0 +1,70 @@
+{
+  include required("../../snippets/picklist-question-multi-list.conf"),
```

Comment:
Isn't this supposed to be single-select ?

Reply:
They want it to be multi-select. See spreadsheet.